README.md

```diff
@@ -26,20 +26,32 @@ After you've read the preparation guide, refer to the deployment example for you
+These are beta versions of 1Password SCIM Bridge deployments and components. These deployments _should_ work, but aren't guaranteed and will change in the future.
 
 - ✨ **NEW** [AWS with CloudFormation](/beta/aws-ecsfargate-cfn)
 - ✨ **NEW** [Azure Container App](/beta/azure-container-apps)
 - [DigitalOcean App Platform with 1Password CLI](/beta/do-app-platform-op-cli/)
 - [Google Workspace settings](/beta/workspace-settings.json)
 - [Google Workspace module for Terraform](/beta/aws-terraform-gw/)
 
-### Deprecated deployments
+## Deprecated deployments
 
-The following are deprecated 1Password SCIM Bridge deployments. At the time of deprecation, these deployments were still fully functional, but may no longer be updated and will eventually be removed.
+A list of recently-deprecated deployments can be found in [`/deprecated`](./deprecated/). At the time of deprecation, these deployments were still fully functional, but will no longer be updated and will be deleted three months after deprecation.
 
-- [AWS EC2 with terraform](/deprecated/aws-terraform/)
-- [DigitalOcean App Platform](/deprecated/digitalocean-app-platform/)
+> 💡 **Note** that it is solely the _deployment method_ that is deprecated. Deprecating a deployment method is independent of the 1Password SCIM Bridge itself, or a specific version of the 1Password SCIM Bridge. For information about the latest 1Password SCIM Bridge version, please see the [changelog](https://app-updates.agilebits.com/product_history/SCIM).
+
+| Deployment                                                           | Deprecation Date | Deletion Date | Suggested Alternative                                                                                                              |
+| -------------------------------------------------------------------- | ---------------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| [aws-ec2-terraform](./deprecated/aws-terraform/)                     | 2020-12-21       | 2023-09-14    | [AWS ECS Fargate with Terraform](./aws-ecsfargate-terraform/) or [AWS ECS Fargate with CloudFormation](./beta/aws-ecsfargate-cfn/) |
+| [DigitalOcean App Platform](./deprecated/digitalocean-app-platform/) | 2022-12-21       | 2023-09-14    | [Digital Ocean App Platform with `op` CLI](./beta/do-app-platform-op-cli/) or [Azure Container Apps](./beta/azure-container-apps/) |
+
+### Deprecation schedule
+
+When a deployment method is deprecated, we will simultaneously post a notice to this README and move the corresponding example to [`/deprecated`](./deprecated/).
+
+Deprecated deployments will remain in [`/deprecated`](./deprecated/) for **three months**, after which time they will be deleted. The deletion date of deprecated deployments will be posted in [`/deprecated/README.md`](./deprecated/README.md).
```

Comment:
Any thoughts on three months? I feel like that's plenty of time for people to clue in and make any required changes if they failed to make their own fork.

Reply:
That sounds fine to me. 

I would personally add a weasel word in front of `three months`, such as about/around/approximately so we can sets dates that suit us, rather than exact periods of time. 

EG: if something gets deprecated on September 25th, let's not delete things on Christmas, but instead at the beginning of the new year.